cms/static/js/models/xblock_info.js

```diff
@@ -172,13 +172,17 @@ function(Backbone, _, str, ModuleUtils) {
             return !this.get('published') || this.get('has_changes');
         },
 
-        canBeDeleted: function(){
+        /**
+         * Return true if section content is required to show.
+         * currently, check section for an entrance exam.
+        */
+        showSectionContent: function(){
```

Comment:
This method name, "showSectionContent" does not make sense-- I don't see how it maps to hiding Delete and Drag.

These should stay as separate methods, and xblocks should indicate their desired behavior so that is_entrance_exam does not need to appear in this file.

For instance, if an xblock should not be deletable, that is the property that should be set instead of "is_entrance_exam".